src/components/WSAudioPlayer.tsx

```diff
@@ -106,6 +106,14 @@ const useStyles = makeStyles((theme: Theme) =>
     },
   })
 );
+const LightTooltip = withStyles((theme) => ({
+  tooltip: {
+    backgroundColor: theme.palette.common.white,
+    color: 'rgba(0, 0, 0, 0.87)',
+    boxShadow: theme.shadows[1],
+    fontSize: 11,
+  },
+}))(Tooltip);
```

Comment:
Could we put LightToolTip in the control folder so we could just have one copy of it?